### DISCUSSION TITLE
combining targets from tar_map() with multiple values

### DISCUSSION THREAD

**Discussion by mkoohafkan** [ropensci/targets#777] [Help]
Consider the following pipeline in `_targets.r`:

```r
library(targets)
library(tarchetypes)

map_spec = expand.grid(number = 1L:3L, category = LETTERS[1:3],
  stringsAsFactors = FALSE)

tar_map(
  values = map_spec,
  names = c("category", "number"),
  tar_target(test, rnorm(1),
    iteration = "list"
  )
)
```

I now want to combine the results of targets within each `category`, e.g., 
- `list(test_A_1, test_A_2, test_A_3)`
- `list(test_B_1, test_B_2, test_B_3)`
- `list(test_C_1, test_C_2, test_C_3)`

Or maybe I instead want to combine within each `number`, e.g.,
- `list(test_A_1, test_B_1, test_C_1)`
- `list(test_A_2, test_B_2, test_C_2)`
- `list(test_A_3, test_B_3, test_C_3)`

Can I use `tar_combine()` to accomplish this?

**Comment by wlandau** [comment]
I think the eval_select() workaround at https://github.com/ropensci/tarchetypes/discussions/82 will help with that.
